.travis.yml

```diff
@@ -32,3 +32,11 @@ script:
   - docker-compose logs
   - curl --retry 10 --retry-delay 5 -D- http://localhost:9200/
   - curl --retry 10 --retry-delay 5 -D- http://localhost:5601/api/status
+  - docker-compose down && docker-compose rm
+
+  # Test docker-swarm.yml
+  - docker swarm init
+  - docker stack deploy -c ./docker-swarm.yml elk
+  - sleep 30
```

Comment:
Is there any way to display the stack logs?

Reply:
Not using a single command, we can display the logs of each services in the stack using the `docker service logs` command though